### PR TITLE
Changing dry-run output to include paths for requirements in make_module_req

### PR DESCRIPTION
**Pull request by rjeschmi**
The output is in 

https://gist.github.com/rjeschmi/5880133e976a8b57a1bf

### CODE REVIEW

File: easybuild/framework/easyblock.py [hunk 1]
```diff
@@ -1069,10 +1071,14 @@ def make_module_req(self):
                 for path in reqs:
                     # only use glob if the string is non-empty
                     if path:
-                        paths = sorted(glob.glob(path))
+                        if self.dry_run:
+                            paths=path
```
Comment (boegel) [inline easybuild/framework/easyblock.py]: style: spaces around `=`

File: easybuild/framework/easyblock.py [hunk 2]
```diff
@@ -1060,7 +1060,9 @@ def make_module_req(self):
                 raise EasyBuildError("Failed to change to %s: %s", self.installdir, err)
 
             lines.append('\n')
+            self.dry_run_msg("List of paths that would be searched and added to modulefile:")
             for key in sorted(requirements):
+                self.dry_run_msg("\t'%s': %s" % (key, requirements[key] ))
```
Comment (boegel) [inline easybuild/framework/easyblock.py]: style, no space before `))`

File: easybuild/framework/easyblock.py [hunk 3]
```diff
@@ -1060,7 +1060,9 @@ def make_module_req(self):
                 raise EasyBuildError("Failed to change to %s: %s", self.installdir, err)
 
             lines.append('\n')
+            self.dry_run_msg("List of paths that would be searched and added to modulefile:")
```
Comment (boegel) [inline easybuild/framework/easyblock.py]: `module file`

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/framework/easyblock.py (hunk 2 above)
Comment (boegel) [inline easybuild/framework/easyblock.py]: add another `self.dry_run_msg('')` to get an empty line between this and the next block in the dry run output

File: easybuild/framework/easyblock.py [hunk 4]
```diff
@@ -1060,7 +1060,13 @@ def make_module_req(self):
...
+
+            if self.dry_run:
+                lines.append(self.module_generator.comment("note: glob patterns are not expanded and existence checks for paths is skipped for the statements below"))
```
Comment (boegel) [inline easybuild/framework/easyblock.py]: line too long

File: easybuild/framework/easyblock.py [hunk 5]
```diff
@@ -1069,11 +1075,16 @@ def make_module_req(self):
                 for path in reqs:
                     # only use glob if the string is non-empty
                     if path:
```
Comment (boegel) [inline easybuild/framework/easyblock.py]: make this `if path and not self.dry_run`?

**Comment by boegel** [comment]
@rjeschmi: some minor open remarks, let's get this merged?

**Comment by boegel** [comment]
@rjeschmi: some enhancements in https://github.com/rjeschmi/easybuild-framework/issues/13, please review/merge, let's get this in :)

**Comment by rjeschmi** [comment]
there is still something off, give me a second. I'm getting output that shouldn't be run in acutal builds.

**Comment by boegel** [comment]
Going in, thanks @rjeschmi!

**Pull request merged**